### PR TITLE
Implement the floor map screen

### PR DESCRIPTION
**Pull request by kubode**
## Issue
- Close https://github.com/DroidKaigi/conference-app-2022/issues/177

## Overview (Required)
- Implements the floor map screen.

### Design

[Figma](https://www.figma.com/file/NcSMs6dMsD88d4wOY0g3rK/DroidKaigi-2022-Conference-App?node-id=1977%3A2763)

## TODO

> The floor map image is decided within 2, 3 days.

Using a dummy image and its URL on this PR.

> The pin icon is for zooming out to see the entire image. This image can be zoomed by pinching.
> I think if you do this issue, you don't have to implement the icon first.

I'll try this feature on the next PR.

### CODE REVIEW

**Comment by takahirom** [comment]
Thanks! Could you check `./gradlew composeLint`?

**Comment by kubode** [comment]
It's should be fixed.
Please check it again 🙏

File: feature/map/src/main/java/io/github/droidkaigi/confsched2022/feature/map/Map.kt [hunk 1]
```diff
@@ -1,30 +1,112 @@
...
+        ) {
+            AsyncImage(
+                model = MapImageUrl,
+                modifier = Modifier
+                    .padding(16.dp)
+                    .fillMaxSize(),
+                contentScale = ContentScale.Fit,
+                alignment = Alignment.Center,
+                contentDescription = "Floor map",
+            )
```
Comment (takahirom) [inline feature/map/src/main/java/io/github/droidkaigi/confsched2022/feature/map/Map.kt:86]: The release date is quite close and may be released as is. Can the icon be commented out for this reason?

**Review by takahirom — APPROVED** [review]
Looks great! Thanks!

File: feature/map/src/main/java/io/github/droidkaigi/confsched2022/feature/map/Map.kt (hunk 1 above)
Comment (kubode) [reply feature/map/src/main/java/io/github/droidkaigi/confsched2022/feature/map/Map.kt:86]: > Can the icon be commented out for this reason?

Do you mean remove the pin icon, which I pointed out in this comment?


I'm going to do this at https://github.com/DroidKaigi/conference-app-2022/issues/688

**Pull request merged**